plugin/trino-phoenix/pom.xml

```diff
@@ -316,12 +336,21 @@
                         <ignoredResourcePattern>assets/org/apache/commons/math3/exception/util/LocalizedFormats_fr.properties</ignoredResourcePattern>
                         <!-- io.airlift:joni and phoenix-client's org.jruby.joni:joni resource duplicates-->
                         <ignoredResourcePattern>tables/.*\.bin</ignoredResourcePattern>
+                        <ignoredResourcePattern>jetty-dir.css</ignoredResourcePattern>
                     </ignoredResourcePatterns>
                     <ignoredDependencies>
                         <dependency>
                             <groupId>com.clearspring.analytics</groupId>
                             <artifactId>stream</artifactId>
                         </dependency>
+                        <dependency>
+                            <groupId>javax.activation</groupId>
```

Comment:
I think it would be preferable to shade these in (or exclude from) the phoenix-client jar.
Maybe the Phoenix project can do that for the next release.

Reply:
The phoenix-client-embedded dependencies in 5.1.0 are just broken.
We will probably release 5.1.1 with the PHOENIX-6377 fix in the next few weeks.

Either wait for that, or manually exclude all depencies of phoenix-client (or perhaps add phoenix-core with provided scope)

Reply:
I did exclude all dependencies from the embedded client, but these were still flagged by maven.